test/verify/check-apps

```diff
@@ -103,12 +102,21 @@ class TestApps(packagelib.PackageCase):
         b.wait_visible(".pf-v5-c-empty-state")
 
         self.createAppStreamPackage("app-1", "1.0", "1")
+        self.createAppStreamPackage("app-2", "1.0", "1", install=True)
         self.createAppStreamRepoPackage()
 
+        b.reload()
+        b.enter_page('/apps')
```

Comment:
This initial reload is ugly and also unnecessary. I see why lines 173 and 183 need it, but nothing here changes os-release or other config in betwen the login_and_go() and here.

Reply:
yea makes sense